products/ruby-on-rails.md

```diff
@@ -81,10 +94,7 @@ releases:
 >[Ruby on Rails](https://rubyonrails.org/), or Rails, is a server-side web application framework
 > written in Ruby.
 
-Only the latest Rails version gets bug fixes, and the version before that gets security fixes.
-Severe security issues (as judged by the core team) are backported further; e.g., v5.2.8.1 is a
-[severe security fix](https://rubyonrails.org/2022/7/12/Rails-Versions-7-0-3-1-6-1-6-1-6-0-5-1-and-5-2-8-1-have-been-released)
-that was created after v5.2 was no longer receiving (non-severe) security updates.
+Each minor release cycle (7.2, 8.0, etc.) is officially supported based on a fixed, yearly duration: 1 year for bug fixes and 2 years for security fixes. For example, if a theoretical 1.1.0 is released on January 1, 2023, it will receive bug and security fixes until January 1, 2024 (1 year), and then receive security fixes only until January 1, 2025 (2 years). After that, it will reach its end-of-life.
```

Comment:
Looks like this policy was update with 7.2: https://guides.rubyonrails.org/v7.2.0/maintenance_policy.html. Maintenance policy for those versions were still the old one: 7.0 (https://guides.rubyonrails.org/v7.0.8/maintenance_policy.html) and 7.1 (https://guides.rubyonrails.org/v7.1.3/maintenance_policy.html). Should we document this as well ? Is there a blog post or a public discussion somewhere that announce this change ?

Reply:
Also found this link, which provides clear dates: https://rubyonrails.org/maintenance. Should we reference this link here ?

Reply:
@marcwrobel I was going by the discussion here https://github.com/rails/rails/pull/52471 and the official page here https://guides.rubyonrails.org/maintenance_policy.html . But it looks like the page you found has even better info. Thanks! I'll update the PR to reference that instead.

Reply:
@marcwrobel I updated this description not clarify that the policy is effective starting with Rails 7.2. I also changed the `releasePolicyLink` value to https://rubyonrails.org/maintenance. That automatically places this link on the rendered page below the description, like this:

> More information is available on the [Ruby on Rails website](https://rubyonrails.org/maintenance).

See c34d8288